docs/reference/changelog-r2023.md

```diff
@@ -0,0 +1,10 @@
+# Webots R2023 Change Log
+
+## Webots R2023a
+Released on XX, XXth, 2023.
+  - New Robots
+  - New Objects
+  - New Features
+  - Enhancements
```

Comment:
I believe this order is slightly better (from the most important to the less important):
```suggestion
  - New Features
  - Enhancements
  - New Robots
  - New Objects
```

Reply:
I kept the order that we have in the current changelog, so we may also want to change it before the release